RealEstate/RealEstate/Pages/Index.cshtml.cs

```diff
@@ -22,31 +22,26 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public void OnGet()
         {
-
-
             using (var webClient = new WebClient())
             {
-                // grab our JSON text. 
-                var houseJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+                var houseurl = "https://data.cityofchicago.org/resource/s6ha-ppgi.json";
+                var landurl = "https://data.cityofchicago.org/resource/aksk-kvfp.json";
+
+                // grab our JSON text.
+                var houseJSON = webClient.DownloadString(houseurl);
 
                 // convert raw text to list.
                 List<Houses> houseCollection = Houses.FromJson(houseJSON);
 
-
                 ViewData["Houses"] = houseCollection;
 
-                var landJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/aksk-kvfp.json");
```

Comment:
The use of a variable instead of direct strings makes it easier to read. Also, when it is referenced multiple times in the bigger codes it is a good practice to use variables instead.
[HTML variables](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/var)